src/main/java/seedu/address/logic/commands/RemindCommand.java

```diff
@@ -78,8 +79,10 @@ private static Assignment createRemindedAssignment(Assignment assignmentToRemind
         ModuleCode updatedModuleCode = assignmentToRemind.getModuleCode();
         Remind updatedRemind = assignmentToRemind.getRemind().setReminder();
         Schedule updatedSchedule = assignmentToRemind.getSchedule();
+        Priority priority = assignmentToRemind.getPriority();
```

Comment:
Perhaps priority can be named as updatedPriority for consistency? But current one is still fine!